docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -604,6 +633,26 @@ $ kubectl create -f alluxio-fuse-client.yaml
 If using the template, Alluxio is mounted at `/alluxio-fuse` and can be accessed via the POSIX-API
 across multiple containers.
 
+***Using `helm`***
+You can deploy the FUSE daemon by configuring the following properties:
+```properties
+fuse:
+  enabled: true
+  clientEnabled: true
+```
+
+Then follow the steps to install Alluxio with helm [here]({{ '/en/deploy/Running-Alluxio-On-Kubernetes.html#deploy-using-helm' | relativize_url }}).
+
+If Alluxio has already been deployed with helm and now you want to enable FUSE, you can manually add the FUSE daemons.
+```console
+# Generate YAML resources with helm template
+$ helm template --name alluxio helm-chart/alluxio/ --set fuse.enabled=true -x templates/fuse/daemonset.yaml -f ./config.yaml > "alluxio-fuse.yaml"
```

Comment:
can we suggest `helm upgrade`? i want to eliminate `helm template` because of the same reason as for the format job

Reply:
Updated to using `helm upgrade`